### DISCUSSION TITLE
Gemini through Vertex AI API

### DISCUSSION THREAD

**Discussion by vadymc** [danny-avila/LibreChat#1829] [Feature Requests & Suggestions]
Hi! First of all, thanks for your work and for the amazing quality product! 
My question is if it's on the roadmap to support Gemini access through Vertex AI API? As far as I can tell [no EU countries](https://ai.google.dev/available_regions) have at the moment access to the Google AI Studio and direct Gemini API (with the API key), so Vertex API seems to be the only possible alternative.
Thanks!

**Comment by danny-avila** [comment]
Thanks for the discussion post! Yes it’s planned and will work on it soon
